lib/code_owners.rb

```diff
@@ -13,14 +13,17 @@ class << self
     # -v -> verbose, outputs details about the matching pattern (if any) for each given pathname
     # -n -> non-matching, shows given paths which don't match any pattern
 
-    def ownerships
-      patterns, owners = pattern_owners.transpose
+    def log(message)
+      warn message
+    end
 
-      git_owner_info(patterns).map do |line, pattern, file|
+    def ownerships
+      patterns = pattern_owners
```

Comment:
admittedly `.transpose` was not very clear, but variable name here is a little misleading... maybe rename the method to `parse_pattern_owners` or `get_pattern_owners`?